asv_bench/benchmarks/rolling.py

```diff
@@ -150,18 +150,16 @@ def time_quantile(self, constructor, window, dtype, percentile, interpolation):
         self.roll.quantile(percentile, interpolation=interpolation)
 
 
-class PeakMemFixed:
+class PeakMemFixedWindowMinMax:
     def setup(self):
-        N = 10
+        N = int(1e5)
         arr = 100 * np.random.random(N)
-        self.roll = pd.Series(arr).rolling(10)
+        self.roll = pd.Series(arr).rolling(1000)
 
     def peakmem_fixed(self):
```

Comment:
can you also add one for min(you can parameterize)

does this take long to run? can you post the results